featurebyte/sql/tile_registry.py

```diff
@@ -51,7 +51,14 @@ async def execute(self) -> None:
             await self.tile_registry_service.create_document(tile_model)
 
         if self.table_exist:
-            cols = await self.get_table_columns(self.table_name)
+            cols = [
+                c.upper()
+                for c in (
+                    await self._session.list_table_schema(
+                        self.table_name, self._session.database_name, self._session.schema_name
+                    )
+                ).keys()
+            ]
```

Comment:
Should we add test to cover this?

Reply:
The issue can be triggered when using Spark 3.2. There is a work in progress branch using that but there are other unrelated issues that need to be fixed first. This path here is already covered by existing test suite.